### PR TITLE
[Proposal] Add stylesheets structure & linters

### PR DESCRIPTION
**Pull request by JuannFerrari**
### This PR:
- Adds a basic structure for stylesheets _(Following this [article](https://dev.to/seibmacdaniel/how-i-structure-my-scss-projects-to-keep-myself-from-going-insane-3jk0) and the [Atomic Design idea](https://bradfrost.com/blog/post/atomic-web-design/))_
- Adds [Stylelint](https://stylelint.io/) linter for scss files.
- Adds [ESLint](https://eslint.org/) linter for javascript files.

### CODE REVIEW

**Review by martinjaimem — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -9,10 +9,16 @@
     "@rails/activestorage": "^6.0.0",
     "@rails/webpacker": "5.2.1",
     "arctic_admin": "^3.2.0",
+    "eslint-config-standard": "^16.0.2",
```
Comment (santib) [inline package.json:12]: why is eslint a prod dependency and not a dev dependency?

File: package.json [hunk 2]
```diff
@@ -9,10 +9,16 @@
...
     "stimulus": "^2.0.0"
   },
   "version": "0.1.0",
```
Comment (santib) [inline package.json:15]: can we add some `scripts` in the `package.json` to be able to run these linters? Like `yarn lint `https://classic.yarnpkg.com/en/docs/cli/run/

**Review by santib — APPROVED** [review]
Love this ❤️

File: package.json (hunk 1 above)
Comment (JuannFerrari) [reply package.json:12]: My bad, I will change it 👌

File: package.json (hunk 2 above)
Comment (JuannFerrari) [reply package.json:15]: Excellent idea

**Pull request merged**